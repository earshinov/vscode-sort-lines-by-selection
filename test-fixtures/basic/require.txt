var fs = require("fs");

var browserSync = [require("browser-sync");]
var watch = [require("gulp-watch");]
var watchify = [require("watchify");]
var source = [require("vinyl-source-stream");]
var gulp = [require("gulp");]
var browserify = [require("browserify");]
var gutil = [require("gulp-util");]
var tsify = [require("tsify");]
var through2 = [require("through2");]
