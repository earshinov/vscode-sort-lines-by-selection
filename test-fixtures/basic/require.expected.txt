var fs = require("fs");

var browserSync = [require("browser-sync");]
var browserify = [require("browserify");]
var gulp = [require("gulp");]
var gutil = [require("gulp-util");]
var watch = [require("gulp-watch");]
var through2 = [require("through2");]
var tsify = [require("tsify");]
var source = [require("vinyl-source-stream");]
var watchify = [require("watchify");]
